src/main/java/com/dmytrohuk/weborganizer/notes/Note.java

```diff
@@ -0,0 +1,58 @@
+package com.dmytrohuk.weborganizer.notes;
+
+import com.dmytrohuk.weborganizer.users.User;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "notes")
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class Note {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "notes_generator")
+    @SequenceGenerator(name="notes_generator", sequenceName = "notes_seq", allocationSize=1)
+
+    private Long id;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "content")
+    private String content;
+
+    @Column(name = "created_date")
+    private LocalDate createdDate;
+
+
+    @Column(name = "updated_date")
+    private LocalDate updatedDate;
+
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false, insertable=false, updatable=false)
+    @JsonIgnore
+    private User user;
+
+
+    @Column(name = "user_id")
```

Comment:
Again, I am not sure why you need both user and userId

Reply:
Ihor told me that userId needs to be there because we will pass an AuthUser (class that implements UserDetails) object which contains an Id that we've taken from User and then we'll set that id into userId